### PR TITLE
firebird: update to 5.0.0.1306

### PR DESCRIPTION
**Pull request by raedrizqie**

### CODE REVIEW

File: mingw-w64-firebird/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+url="https://www.firebirdsql.org/"
+license=('spdx:Interbase-1.0 AND MPL-1.0')
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64')
+msys2_references=(
+  'archlinux: firebird'
+)
+msys2_repository_url='https://github.com/FirebirdSQL/firebird/'
+msys2_documentation_url='https://www.firebirdsql.org/en/documentation/'
+conflicts=("${MINGW_PACKAGE_PREFIX}-firebird2-git")
```
Comment (Biswa96) [inline mingw-w64-firebird/PKGBUILD:19]: Should firebird2-git be removed?

File: mingw-w64-firebird/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+depends=("${MINGW_PACKAGE_PREFIX}-icu"
+         "${MINGW_PACKAGE_PREFIX}-libiconv"
+         "${MINGW_PACKAGE_PREFIX}-libtomcrypt"
+         "${MINGW_PACKAGE_PREFIX}-libtommath"
+         "${MINGW_PACKAGE_PREFIX}-re2")
+makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
+             "${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-autotools"
+             "unzip")
+options=('!buildflags')
```
Comment (Biswa96) [inline mingw-w64-firebird/PKGBUILD:30]: would you like to explain the reason to disable build flags?

File: mingw-w64-firebird/PKGBUILD (hunk 1 above)
Comment (raedrizqie) [reply mingw-w64-firebird/PKGBUILD:19]: i think so.. v2 is not even working anyway

File: mingw-w64-firebird/PKGBUILD (hunk 2 above)
Comment (raedrizqie) [reply mingw-w64-firebird/PKGBUILD:30]: that can safely be removed imo.. i forgot to remove it

File: mingw-w64-firebird/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+build() {
+  cd ${_realname}-${pkgver%.*}
+
+  if [[ ${MSYSTEM} == CLANG* ]]; then
+    CXXFLAGS="-Wno-varargs -Wno-constant-conversion -Wno-inconsistent-missing-override -Wno-tautological-constant-out-of-range-compare -Wno-c++11-narrowing"
+  else
+    CXXFLAGS="-Wno-class-memaccess -Wno-overflow -Wno-return-local-addr -Wno-stringop-overread -Wno-stringop-overflow -Wno-narrowing"
+  fi
+
+  ./configure \
```
Comment (MehdiChinoune) [inline mingw-w64-firebird/PKGBUILD:95]:
```suggestion
  mkdir -p build-${MSYSTEM} && cd build-${MSYSTEM}

  if [[ ${MSYSTEM} == CLANG* ]]; then
    CXXFLAGS="-Wno-varargs -Wno-constant-conversion -Wno-inconsistent-missing-override -Wno-tautological-constant-out-of-range-compare -Wno-c++11-narrowing"
  else
    CXXFLAGS="-Wno-class-memaccess -Wno-overflow -Wno-return-local-addr -Wno-stringop-overread -Wno-stringop-overflow -Wno-narrowing"
  fi

  ../${_realname}-${pkgver%.*}/configure \
```
Is out-of-tree building possible?

Comment (raedrizqie) [reply mingw-w64-firebird/PKGBUILD:95]: tried once before.. and it failed (something related to the generated source files)

**Comment by raedrizqie** [comment]
@lazka can we merge this?

**Pull request merged**